docs/_docs/Unofficial Sidecar Academy/Sidecar-Academy.md

```diff
@@ -0,0 +1,106 @@
+---
+title: LDSA Unofficial Sidecar Academy
+category: Unofficial Sidecar Academy
+order: 1
+---
+
+Contents:
+- [Course overview](#course-overview)
+- [Structure](#structure)
+- [Time Commitments](#time-commitments)
+- [What will be covered](#what-will-be-covered)
+- [What is the same and what is different](#what-is-the-same-and-what-is-different)
+- [Introduction session](#introduction-session)
+- [Hackathons](#hackathons)
+- [Schedule](#schedule)
+- [Late policy](#late-policy)
+- [Frequently Asked Questions (FAQ)](#faq)
```

Comment:
Link does not work, needs section name. The following syntax should work:
`[Frequently Asked Questions (FAQ)](#frequently-asked-questions-faq)`